doc/doxygen/install/install-linux.doxygen

```diff
@@ -109,7 +109,7 @@
     sudo add-apt-repository universe
     sudo apt update
     sudo apt-get install build-essential cmake autoconf patch libtool git automake
-    sudo apt-get install qtbase5-dev libqt5svg5-dev libqt5opengl5-dev
+    sudo apt-get install qtbase6-dev libqt6svg6-dev libqt6opengl6-dev
```

Comment:
Hehe, you wish!
I am sorry to disappoint you but at least for ubuntu, the package structure changed quite a bit. See my changes in the GitHub Actions. Also a standalone opengl implementation is needed now.